operator/v1/types_network.go

```diff
@@ -408,3 +423,22 @@ const (
 	// IPAMTypeStatic uses static IP
 	IPAMTypeStatic IPAMType = "Static"
 )
+
+// TcPolicy define the TC Policy of OVS Hardware Offload
+type TcPolicy string
```

Comment:
Why not full name: TrafficControlPolicy?

Reply:
Updated with full name: TrafficControlPolicy.